src/exception.rs

```diff
@@ -103,6 +103,11 @@ pub fn handle_exception_sync(ctx: &mut TrapFrame) -> AxResult<AxVCpuExitReason>
             // The `#imm`` argument when triggering a hvc call, currently not used.
             let _hvc_arg_imm16 = ESR_EL2.read(ESR_EL2::ISS);
 
+            // Is this a psci call?
```

Comment:
Add more comments here please.

Expecially for ArceOS, when its platform is qemu-virt-aarch64, it use hvc to trigger `psci_call`

Reply:
OK

Reply:
Added